ts/mains/main_node.ts

```diff
@@ -995,6 +995,28 @@ ipc.on('password-window-login', async (event, passPhrase) => {
   }
 });
 
+ipc.on('password-recovery-phrase', async (event, passPhrase) => {
```

Comment:
Did this exist before onboarding and I somehow removed it by mistake? It looks familar 🤔

Reply:
no it didn't. It's brand new